OrbitGl/TimeGraph.h

```diff
@@ -107,7 +108,8 @@ class TimeGraph {
   void SetCanvas(GlCanvas* canvas);
   [[nodiscard]] GlCanvas* GetCanvas() { return canvas_; }
   void SetFontSize(int font_size);
-  [[nodiscard]] int GetFontSize() { return GetTextRenderer()->GetFontSize(); }
+  [[nodiscard]] int GetFontSize() { return lround((GParams.font_size) * layout_.GetScale()); }
+  [[nodiscard]] int GetOverlayFontSize() { return lround(GParams.font_size); }
```

Comment:
These methods should be const and the return type should probably be uint32_t. I know that int was used before, but since we're here... :-)